src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/ToggleSwitchSamplePage.xaml

```diff
@@ -169,6 +169,22 @@
 							<ToggleSwitch Header="HorizontalAlignment=Stretch"
 										  HorizontalAlignment="Stretch"
 										  Style="{StaticResource MaterialToggleSwitchStyle}" />
+
+							<!-- ToggleSwitch ON change foreground -->
```

Comment:
@kazo0 We should probably decide on a place to hide (while still being accessible) these specific examples that showcase failure points. But they definitely dont belong on the sample itself.

Reply:
Like this would be ideal for ui-tests, or runtimes tests(RTT with screenshots has limited platforms support?).

Reply:
yeah makes sense, the ui tests are able to navigate to the "nested" pages directly so we can probably take advantage of that

Reply:
to be pedantic, "nested" pages are just pages not indexed when building the items-source for the navigation, but still public accessible to regular user through navigation
we should have some less accessible pages for these kind of "tests" pages